tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWZ-5789-specs.js

```diff
@@ -0,0 +1,45 @@
+const Page = require('../PopupTabLayoutPage');
+const {getFocusedText} = require('../../utils');
+
+describe('qa-sampler', function () {
+	const {popupTabLayout} = Page.components;
+
+	beforeEach(async function () {
+		await Page.open('WithVariousItems', '?locale=ar-SA');
+	});
+
+	describe('PopupTabLayout', function () {
+		it('should not navigate with left and right key in RTL - [QWZ-5789]', async function () {
```

Comment:
we don't use QWZ project.
```suggestion
		it('should not navigate with left and right key in RTL - [QWTC-2649]', async function () {
```